src/main/kotlin/com/atlassian/performance/tools/report/api/JdkComparisonPlaintextReport.kt

```diff
@@ -0,0 +1,57 @@
+package com.atlassian.performance.tools.report.api
+
+import com.atlassian.performance.tools.jiraactions.api.ActionMetricStatistics
+import org.apache.commons.lang3.StringUtils
+import java.lang.StringBuilder
+import java.time.Duration
+import java.util.Formatter
+
+class JdkComparisonPlaintextReport(
+    val jdk8Statistics: ActionMetricStatistics,
+    val jdk11Statistics: ActionMetricStatistics
+) {
+    fun generate(): String {
+        val percentile = 95
+        val jdk8Percentile = jdk8Statistics.percentile(percentile)
+        val jdk11Percentile = jdk11Statistics.percentile(percentile)
+
+        val report = StringBuilder()
+        val formatter = Formatter(report)
+        val lineFormat = "| %-25s | %-29d | %-28d | %-25d | %-9s |\n"
+
+        formatter.format("\n")
+        formatter.format("+---------------------------+-------------------------------+------------------------------+---------------------------+-----------+\n")
+        formatter.format("| Action name               | JDK 11 - 95th percentile [ms] | JDK 8 - 95th percentile [ms] | Diff 95th percentile [ms] | Outcome   |\n")
+        formatter.format("+---------------------------+-------------------------------+------------------------------+---------------------------|-----------+\n")
+
+        jdk8Statistics
+            .sampleSize
+            .keys
+            .sorted()
+            .forEach { action ->
+                val diff = compare(jdk8Percentile[action], jdk11Percentile[action])
+                formatter.format(
+                    lineFormat,
+                    StringUtils.abbreviate(action, 25),
+                    jdk11Percentile[action]?.toMillis(),
+                    jdk8Percentile[action]?.toMillis(),
```

Comment:
If you'd like to compare JDK 9, JDK 12 or other JDK distributions (Adopt, Oracle, Open, Corretto), they wouldn't be covered. A person interested in these results would have to wait for the feature to be written, merged in and released. It's a design [closed for extension](https://en.wikipedia.org/wiki/Open%E2%80%93closed_principle).
Current `DataReporter` has open design, it will compare any cohorts. `JdkComparisonPlaintextReport` would have an advantage over `DataReporter` if it had some JDK-specific logic and insights.